contrib/docker1/reference/digestset/set.go

```diff
@@ -0,0 +1,279 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+/*
+   Copyright The docker/distribution Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This redundancy is for making ltag happy.

I think ideally we should just list a single Apache boilerplate with `Copyright The docker/distribution Authors and containerd Authors`.

@kunalkushwaha